react/javascript/file-download/package.json

```diff
@@ -20,11 +20,16 @@
     "node": ">=12"
   },
   "dependencies": {
-    "@looker/components": "^0.15.1",
-    "@looker/extension-sdk": "^21.0.8",
-    "@looker/extension-sdk-react": "^21.0.8",
-    "@looker/sdk": "^21.0.8",
-    "@looker/sdk-rtl": "^21.0.8",
+    "@looker/components": "^1.1.6",
+    "@looker/icons": "1.1.5",
+    "@looker/embed-sdk": "^1.6.0",
+    "@looker/extension-sdk": "^21.4.5",
+    "@looker/extension-sdk-react": "^21.4.5",
+    "@looker/sdk": "^21.4.3",
```

Comment:
Do you want 21.6.0? Or not critical

Reply:
At the moment not urgent. Next
1. lint
2. test (very simple render test per project, will add more later)
3. lerna or workspaces to have a single build
4. build on PR create (git action)
5. use Izzy's script to update package versions

Hope to wrap up lint and get one test going by the end of the day